src/Symfony/Component/Serializer/NameConverter/NameConverterInterface.php

```diff
@@ -21,16 +21,20 @@
     /**
      * Converts a property name to its normalized value.
      *
-     * @param string $propertyName
+     * @param string|object $class
+     * @param string        $propertyName
+     *
      * @return string
      */
-    public function normalize($propertyName);
+    public function normalize($class, $propertyName);
```

Comment:
Well, it says ``Should be avoided``, because it is still a BC break. The ``Regular`` column is a column saying "We don't actually provide BC here, but only partial BC".

And actually, I don't think any code added since 2.3 has actually been tagged as ``@api``. Many things are extension points for our users where we care about BC as if they were flagged as ``@api`` even though it was not done.
Given that we became better at writing BC layers than a few years back, we may decide to drop the distinction between Regular and API btw. The default behavior of a class should be full BC. The meaningful distinction would be Regular vs Internal (where internal classes not meant to be used by users are less strict on BC)